src/components/views/room_settings/RoomProfileSettings.tsx

```diff
@@ -270,8 +271,10 @@ export default class RoomProfileSettings extends React.Component<IProps, IState>
                             disabled={!this.state.canSetName}
                         />
                         <Field
-                            className="mx_ProfileSettings_controls_topic"
-                            id="profileTopic"
```

Comment:
Why remove the `id`?

Reply:
Ah right, this is used for `NewRoomIntro` and should not be removed. Thanks!